apps/meteor/client/sidebarv2/RoomList/RoomList.tsx

```diff
@@ -66,14 +44,27 @@ const RoomList = () => {
 	usePreventDefault(ref);
 	useShortcutOpenMenu(ref);
 
-	const { groupCounts, groupList, roomList } = getRoomsByGroup(roomsList);
+	const handleCollapse = (groupTitle: string) => {
+		if (groupTitle) {
+			handleCollapsedGroups(groupTitle);
+		}
+	};
 
 	return (
 		<Box position='relative' display='flex' overflow='hidden' height='full' flexGrow={1} flexShrink={1} flexBasis='auto' ref={ref}>
 			<GroupedVirtuoso
-				groupCounts={groupCounts}
-				groupContent={(index) => <SidebarV2GroupTitle title={t(groupList[index])} />}
-				itemContent={(index) => <RoomListRow data={itemData} item={roomList[index]} />}
+				groupCounts={groupsCount}
+				groupContent={(index) => (
+					<SidebarV2CollapseGroup
```

Comment:
Refreshing the page with a group open seem to leave a gap after the section, even when the group is collapsed. It fixes itself if you collapse the group and refresh again.

![Screenshot 2024-10-18 at 12 02 50](https://github.com/user-attachments/assets/6c2cb814-b58c-4883-aadf-5cd3df6b8675)

Reply:
did you run `yarn && yarn build`? it seems to me you're not running with the correct fuselage version

Reply:
Yep, that solved it 🚀